web_src/less/_base.less

```diff
@@ -893,13 +893,13 @@ a.ui.card:hover,
     }
 
     &.blue {
-      color: var(--color-primary) !important;
+      color: var(--color-blue) !important;
 
       a {
         color: inherit !important;
 
         &:hover {
-          color: var(--color-primary-dark-1) !important;
+          color: #3876b3 !important;
```

Comment:
There probably need to be --color-blue-dark etc.

Reply:
Yes, we don't have light/dark variant colors yet. Probably something for another PR.

Reply:
Actually, I added just this one color now.

Reply:
I think it's not dark, but light .....

`hover` usually means light color IMO

Reply:
There is a similar color above (red), `hover` is light color, not `dark`


![image](https://user-images.githubusercontent.com/2114189/169395984-3cc86d88-b35c-44fc-b417-0b0f45425ac1.png)

Reply:
Right, light/dark apply to default theme only, it's inverted on dark themes.